README.md

```diff
@@ -3,53 +3,77 @@
 ScratchCloud is a new, powerful, Scratch cloud data API for Node.js written by AmazingMech2418! 
 
 ## Features
-- Scratch login session handling
+- Scratch login session handling (no logouts)
```

Comment:
There are logouts.

Reply:
I don't seem to get logged out anymore though.

Reply:
@AmazingMech2418

Reply:
@AmazingMech2418